### PR TITLE
Update TX committees

### PR DESCRIPTION
**Pull request by alexobaseki**
@NewAgeAirbender  Check this out if, it is missing anything.

### CODE REVIEW

**Comment by NewAgeAirbender** [comment]
So far I have two notes: 

1. Committees that don't exist any longer can be deleted instead of moved to `retired`
2. Is there a way to strip the jurisdiction & chamber from the Committee name? Most jurisdictions don't list either in the name & will lead to issues matching committees to actions on Plural's end but also issues matching them to Events/Votes/Sponsorship on OS if they're not similar. I think we can discuss & plan to edit or update either side

**Comment by alexobaseki** [comment]
> So far I have three notes:
> 
> 1. Committees that don't exist any longer can be deleted instead of moved to `retired`
> 2. Is there a way to strip the jurisdiction & chamber from the Committee name? Most jurisdictions don't list either in the name & will lead to issues matching committees to actions on Plural's end but also issues matching them to Events/Votes/Sponsorship on OS if they're not similar. Could add it as an `other_name` to the yaml file if it can't be edited. I think we can discuss & plan to edit or update either side.
> 3. If a member exists on a committee without being a legislator, we just add them with the `role: member` and don't set the `person_id` at all so doesn't need to add a `null` value.



> So far I have three notes:
> 
> 1. Committees that don't exist any longer can be deleted instead of moved to `retired`
> 2. Is there a way to strip the jurisdiction & chamber from the Committee name? Most jurisdictions don't list either in the name & will lead to issues matching committees to actions on Plural's end but also issues matching them to Events/Votes/Sponsorship on OS if they're not similar. Could add it as an `other_name` to the yaml file if it can't be edited. I think we can discuss & plan to edit or update either side.
> 3. If a member exists on a committee without being a legislator, we just add them with the `role: member` and don't set the `person_id` at all so doesn't need to add a `null` value.



> So far I have three notes:
> 
> 1. Committees that don't exist any longer can be deleted instead of moved to `retired`
> 2. Is there a way to strip the jurisdiction & chamber from the Committee name? Most jurisdictions don't list either in the name & will lead to issues matching committees to actions on Plural's end but also issues matching them to Events/Votes/Sponsorship on OS if they're not similar. Could add it as an `other_name` to the yaml file if it can't be edited. I think we can discuss & plan to edit or update either side.
> 3. If a member exists on a committee without being a legislator, we just add them with the `role: member` and don't set the `person_id` at all so doesn't need to add a `null` value.

Made these changes, please take a look if they make sense.

File: data/tx/committees/lower-Educational-Opportunity--Enrichment-Select-22fcc747-bc8d-4a3f-b610-65c766fa1beb.yml [hunk 1]
```diff
@@ -1,63 +1,64 @@
...
-- url: https://capitol.texas.gov/Committees/MeetingsByCmte.aspx?Leg=88&Chamber=H&CmteCode=C232
+- url: https://www.house.texas.gov/committees/committee/?committee=C232
 members:
 - name: Brad Buckley
-  role: chair
+  role: 'chair '
   person_id: ocd-person/e3533cbc-f3ab-46bc-a591-c68657f82527
 - name: Barbara Gervin-Hawkins
-  role: vice chair
+  role: 'vice chair '
```
Comment (NewAgeAirbender) [inline data/tx/committees/lower-Educational-Opportunity--Enrichment-Select-22fcc747-bc8d-4a3f-b610-65c766fa1beb.yml:21]: Any chance we can just have this as a striped non-string? Noticed when `chair` or `vice chair` get added it's as 'chair ' & 'vice chair ', don't think it really matters but just curious

**Review by NewAgeAirbender — APPROVED** [review]
Have one that's not necessary but the rest looks good!

File: data/tx/committees/lower-Educational-Opportunity--Enrichment-Select-22fcc747-bc8d-4a3f-b610-65c766fa1beb.yml (hunk 1 above)
Comment (alexobaseki) [reply data/tx/committees/lower-Educational-Opportunity--Enrichment-Select-22fcc747-bc8d-4a3f-b610-65c766fa1beb.yml:21]: Good catch

Comment (alexobaseki) [reply data/tx/committees/lower-Educational-Opportunity--Enrichment-Select-22fcc747-bc8d-4a3f-b610-65c766fa1beb.yml]: Updated

**Pull request merged**